azure/packages/test/end-to-end-tests/src/test/containerCopy.spec.ts

```diff
@@ -115,16 +115,16 @@ describe("Container copy scenarios", () => {
 	});
 
 	/**
-	 * Scenario: test if Azure Client can copy existing container at specific version.
+	 * Scenario: test if Azure Client can copy existing container at a specific version.
 	 *
 	 * Expected behavior: an error should not be thrown nor should a rejected promise
 	 * be returned.
 	 */
-	it.skip("TEST FAILING SO SKIPPED - can sucesfully copy document from a specific version", async () => {
+	it("can successfully copy an existing container at a specific version", async () => {
 		const { container } = await client.createContainer(schema);
 		const containerId = await container.attach();
 
-		await timeoutPromise((resolve) => container.once("connected", () => resolve()), {
+		await timeoutPromise((resolve) => container.once("connected", resolve), {
```

Comment:
Technically better to leave this as-is - if the event params change on the "connected" event then the type of this promise implicitly changes as well (resolve will be called with the event args), whereas we strictly want a void promise here.  Same for the one below.